code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -374,7 +374,7 @@
 	. += SPAN_NOTICE("It is currently [zipped ? "zipped" : "unzipped"]. Alt+Shift+Click to [zipped ? "un-" : ""]zip it!")
 
 /obj/item/storage/backpack/duffel/AltShiftClick(mob/user)
-	. = ..()
+	visible_message(SPAN_WARNING("[user] tries to [zipped ? "un-" : ""]zip [src]."), SPAN_WARNING("You start to [zipped ? "un-" : ""]zip [src]."))
```

Comment:
I would recommend moving this to `handle_zipping()` proc, below the adjacency check.
That is because, unless I am mistaken, putting it here without any checks will send this message to anyone who Alt-Shift-Clicks a duffelbag, even one lying on the other side of the room.
Also, `SPAN_WARNING` is used primarily for dangerous actions if I am not mistaken. `SPAN_NOTICE` or something similar may be more appropriate.